wrangler-core/src/main/antlr4/io/cdap/wrangler/parser/Directives.g4

```diff
@@ -124,7 +124,7 @@ properties
  ;
 
 propertyList
- : property (',' property)+
+ : property (',' property)*
```

Comment:
Rebase this branch onto latest develop with this change

Reply:
Done